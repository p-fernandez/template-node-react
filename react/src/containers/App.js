import React from 'react';
import styled from 'styled-components';

const AppLayout = styled.div`
  height: 100vh;
  margin: 0;
  padding: 0;
  width: 100%;
`;

const App = (props) => (
  <AppLayout className="appLayout">
    {props.children}
  </AppLayout>
);

export default App;
