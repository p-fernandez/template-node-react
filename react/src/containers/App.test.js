import React from 'react';
import renderer from 'react-test-renderer';

import App from './App';

const getComponent = () => renderer.create(<App />);

describe('App', () => {
  test('Should render properly', () => {
    expect(getComponent()).toMatchSnapshot();
  })
});
