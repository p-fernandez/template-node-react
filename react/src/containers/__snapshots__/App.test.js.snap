// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Should render properly 1`] = `
<div
  className="sc-bdfBwQ iWkkKT appLayout"
/>
`;
